src/debug/ee/arm64/arm64walker.cpp

```diff
@@ -125,8 +123,9 @@ BYTE*  NativeWalker::SetupOrSimulateInstructionForPatchSkip(T_CONTEXT * context,
 
        
     }
-    else if ((opcode & 0xFF000000) == 0x58000000) //Load register (literal)
-    {
+    
+    else if ((opcode & 0x3BF000000) == 0x18000000) //Load register (literal)
+     {
```

Comment:
This should be:

if ((opcode & 0x3B0000000) == 0x18000000) //LDR Literal (General or SIMD)